R-package/tests/testthat/test_callbacks.R

```diff
@@ -236,7 +238,7 @@ test_that("early stopping xgb.train works", {
 test_that("early stopping using a specific metric works", {
   set.seed(11)
   expect_output(
-    bst <- xgb.train(param, dtrain, nrounds = 20, watchlist, eta = 0.6,
+    bst <- xgb.train(param[-2], dtrain, nrounds = 20, watchlist, eta = 0.6,
```

Comment:
What is this for? @lorentzenchr

Reply:
param[2] sets eval_metric. I exclude it here, because it is already specified by kwarg. Otherwise it throws a warnung and the test fails. 

Should I add a comment or alternatively remove the kwarg?

Reply:
Got it. Let's keep it there.